packages/amesos2/src/Amesos2_ShyLUBasker_def.hpp

```diff
@@ -65,7 +65,8 @@ ShyLUBasker<Matrix,Vector>::ShyLUBasker(
   ShyLUbasker->Options.use_nodeNDP       = BASKER_TRUE;  // use nodeNDP to compute ND partition
   ShyLUbasker->Options.run_nd_on_leaves  = BASKER_TRUE;  // run ND on the final leaf-nodes
   ShyLUbasker->Options.run_amd_on_leaves = BASKER_FALSE; // run AMD on the final leaf-nodes
-  ShyLUbasker->Options.transpose     = BASKER_FALSE;
+  ShyLUbasker->Options.transpose      = BASKER_FALSE;
+  ShyLUbasker->Options.threaded_solve = BASKER_FALSE;
```

Comment:
@iyamazaki you'll need to add the `small_matrix` option to Amesos2 at some point, that can be in a follow on PR if you prefer